CHANGES.rst

```diff
@@ -1595,10 +1595,10 @@ astropy.coordinates
 
 astropy.cosmology
 ^^^^^^^^^^^^^^^^^
-- The pre-publication Planck 2018 cosmological parameters are included as the
-  ``Planck2018_arXiv_v2`` object.  Please note that the values are preliminary,
-  and when the paper is accepted a final version will be included as
-  ``Planck18``. [#8111]
+- The final version of the Planck 2018 cosmological parameters are included
```

Comment:
Instead of modifying change log of an older release, this should be a new change log entry under 4.2.

Reply:
Oh, right, sorry, missed the section heading.